core/src/main/java/org/mskcc/cbio/maf/MafUtil.java

```diff
@@ -560,7 +560,11 @@ public MafRecord parseRecord(String line)
 		record.setDriverFilter(TabDelimitedFileUtil.getPartStringAllowEmpty(driverIndex, parts));
 	    }
 	    record.setDriverFilterAnn(TabDelimitedFileUtil.getPartStringAllowEmpty(driverAnnIndex, parts));
-	    record.setDriverTiersFilter(TabDelimitedFileUtil.getPartStringAllowEmpty(driverTiersIndex, parts));
+	    if (driverTiersIndex != -1 && parts[driverTiersIndex].trim().equals("NA")) {
```

Comment:
you could write this as:
`if (TabDelimitedFileUtil.getPartStringAllowEmpty(driverTiersIndex, parts).equals("NA"))`

or better write a method `TabDelimitedFileUtil.getPartStringAllowEmptyAndNA` and just use that.
```java
	public static String getPartStringAllowEmptyAndNA(int index, String[] parts)
	{
		String value = getPartStringAllowEmpty(index, parts);
		if (value.equals(NA_STRING)) {
				value = "";
		}
		return value;
}
```